content/foundations/color/accessibility.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Accessibility
+description:
+  Primer color tokens are designed to be accessible by default. This guide provides more context around how to ensure new patterns meet GitHub's standards.
+---
+
+import {Box} from '@primer/react'
+
+## Guidelines
+
+### Assure adequate color contrast
+
+Color contrast between text and its background must meet [required WCAG standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).
```

Comment:
WCAG [recommends](https://www.w3.org/TR/WCAG22/) to use WCAG22 standard. It's the same link but with `22` -> https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html.

Might be worth to ask Tetralogical for review as they are working on a11y documentation at the moment (cc @alliethu)

Reply:
Thanks! This is all copy/paste from the original color docs. Let's take the time to get it right even if we're just moving it to a new page 👍